README.md

```diff
@@ -197,7 +197,7 @@ ALTER USER EXAMPLE_USER SET RSA_PUBLIC_KEY = 'MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMI
 ```
 5. Verify your auth setup. If you have `snowsql` installed, that has an easy method (CTRL-d to exit)
 ```bash
-snowsql -a <account_identifier>.<region>p -u <user> --private-key-path private_key.pem
+snowsql -a <account_identifier>.<region> -u <user> --private-key-path private_key.pem
```

Comment:
just checking: "account_identifier" is `<orgname>.<account_name>`, is that right?
and "region" would be in a format like `us-east-1`?
would this be a good example?
```suggestion
# example: snowsql -a AAAAAAA.BBBBBBBB.us-east-1 -u john --private-key-path private_key.pem
snowsql -a <account_identifier>.<region> -u <user> --private-key-path private_key.pem
```